relay-config/src/config.rs

```diff
@@ -821,6 +821,8 @@ pub struct Spool {
 #[derive(Serialize, Deserialize, Debug)]
 #[serde(default)]
 struct Cache {
+    /// The full project state will be requested by this Relay if set to `true`.
+    project_state_full: bool,
```

Comment:
nit: Should we call this `project_request_full_config` or something similar?

Reply:
@jjbayer  I like it. Renamed.